package.json

```diff
@@ -4,17 +4,19 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run build && webpack --config ./config/webpack.config.js && npm run test:unit && npm run test:e2e",
-    "test:bamboo": "npm run build && webpack --config ./config/webpack.config.js && npm run test:unit:bamboo && npm run test:e2e:bamboo",
+    "test": "npm run build && npm run test:unit && npm run test:e2e",
+    "test:bamboo": "npm run build && npm run test:e2e:bamboo && npm run test:unit:bamboo",
     "test:server:start": "http-server ./ &",
     "test:unit": "wct --config-file ./config/wct.local.config.json",
     "test:unit:bamboo": "wct --config-file ./config/wct.docker.config.json",
-    "test:e2e": "npm run test:server:start && mocha --chrome --config ./config/mocha.config.json --require esm",
-    "test:e2e:bamboo": "npm run test:server:start && npm run test:e2e:bamboo:chrome && npm run test:e2e:bamboo:firefox && npm run test:e2e:bamboo:edge",
+    "test:e2e": "build-storybook && npm run test:server:start && mocha --chrome --config ./config/mocha.config.json --require esm",
```

Comment:
Ik kan mis zijn maar moet dit niet zijn `npm run build-storybook`?

Reply:
bij `npm run build-storybook` gaat hij eerst builden en dan de storybook builden (zie script). Omdat die testen al in een chain zitten moet er niet opnieuw gebuild worden.

Reply:
Ok :) just checking.